.travis/build-site.sh

```diff
@@ -44,20 +32,7 @@ fi
 
 (
     if [[ "$VERSION" == *-SNAPSHOT && "$TRAVIS_BRANCH" == "master" ]]; then
-        # this can take very long and no output is generated. therefore use the background job again
-        export PING_SLEEP=30s
-        export BUILD_OUTPUT=/tmp/build-site-upload.out
-        export PING_PID_FILE=/tmp/build-site-upload-ping.pid
-
-        source .travis/background-job-funcs.sh
-
         # Uploading snapshot site...
-        rsync -ah --stats --delete target/pmd-doc-${VERSION}/ ${PMD_SF_USER}@web.sourceforge.net:/home/project-web/pmd/htdocs/snapshot/
-
-        # The build finished without returning an error so dump a tail of the output
-        dump_output
-
-        # nicely terminate the ping output loop
-        kill_ping
+        travis_wait rsync -ah --stats --delete target/pmd-doc-${VERSION}/ ${PMD_SF_USER}@web.sourceforge.net:/home/project-web/pmd/htdocs/snapshot/
```

Comment:
I once tried using it, but it failed:
https://github.com/pmd/pmd/commit/cd39cd6fa8486fef01811725c3d303e9accc2fab
https://travis-ci.org/pmd/pmd/jobs/241675415

~~~
.travis/build-site.sh: line 50: travis_wait: command not found
~~~

Seems like travis_wait isn't exported to the subshell, in which we run build-site.sh... using a login-shell (which loads bashrc) didn't help either: https://github.com/pmd/pmd/commit/929afd8318314a53b561dfc4f7ea5ef4769026ed now, we got the wrong java version... maybe a "export travis_wait" would work?

Btw - travis_wait is defined here: https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/templates/header.sh